components/CovidMap.js

```diff
@@ -0,0 +1,702 @@
+}
+
+const clusterStyleSecondLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleThirdSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-third'
+}
+
+const formStyle = {
+  width: '50%',
+  float: 'left', 
+}
+
+const legendStyle = {
+  width: '50%', 
+  float: 'right',
+}
+
+const legendItemStyle = {
+  width: '100%',
+  marginBottom: '5px',
+}
+
+const legendTextStyle = {
+  width: '90%',
+  height: '20px',
+  float: 'left',
+  textAlign: 'right',
+}
+
+const legendIconStyle = {
+  minWidth: '30px',
+  height: '20px',
+  float: 'right',
+  display: 'inline-block',
+}
+
+const legendIconFirst = {
+  background: '#0B5D93',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'right',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconSecond = {
+  background: '#634562',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconThird = {
+  backgroundColor: '#CE2727',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+
+class Map extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      map: null,
+      firstClusterer: null,
+      secondClusterer: null, 
+      thirdClusterer: null,
+      fetchedMap: false,
+      data: [],
+      selectUpdate: true,
+      selectedOption: props.value || 'all',
+      firstLegendText: 'All Deaths',
+      secondLegendText: '',
+      thirdLegendText: ''
+    }
+  }
+
+  getGoogleMaps() {
+    // If we haven't already defined the promise, define it
+    if (!this.googleMapsPromise) {
+      this.googleMapsPromise = new Promise((resolve) => {
+        // Add a global handler for when the API finishes loading
+        window.resolveGoogleMapsPromise = () => {
+          // Resolve the promise
+          const google = window.google;
+          resolve(google);
+
+          // Tidy up
+          delete window.resolveGoogleMapsPromise;
+        };
+
+        // Load the Google Maps API
+        const API = 'AIzaSyDAh7M89BnID8kGVXBrNtxJfD-jjDDFRCg';
+        var script = document.createElement("script");
+        script.src = `https://maps.googleapis.com/maps/api/js?key=${API}&callback=resolveGoogleMapsPromise`;
+        script.async = true;
+        document.body.appendChild(script);
+        script.src = "https://googlemaps.github.io/v3-utility-library/packages/markerclustererplus/dist/markerclustererplus.min.js";
```

Comment:
Do we need this reference if we're importing `@google/markerclustererplus` into this file?

Reply:
Looks like we don't need it! I just removed in and tested it locally, I think it's safe to remove these lines.